.changeset/cuddly-houses-share.md

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
From the video I watched - my understanding is that when we are ready to public a new release, in that PR these files will be deleted and the versions bumped.

Reply:
Yes. I'm going to merge this PR now and the bot should open a bump PR in a minute

Reply:
small fix: https://github.com/Eppo-exp/eppo-multiplatform/pull/140 🙂

Reply:
here: https://github.com/Eppo-exp/eppo-multiplatform/pull/141